docs/segment.md

```diff
@@ -2,98 +2,188 @@
+
+![image: Labeling a text region ](img/segment/labeling_text_regions.gif "We click on a text region and label it with a label from the text region ontology.")
 
-![image: Screenshot of the 5 activated panels (Segmentation panel framed in red) ](img/segment/segment_panel.png "The 3d panel is used to manually edit the segmentation")
+!!! tip
+    You can change the color of a text region in eScriptorium in the **images** tab by clicking on **editor settings**. 
 
-## Overview of the segmentation panel
+!!! note
+    It is possible to create nested text regions, just draw another bounding box on top of the text regions you want to annotate!
 
-<!-- rewrite this section-->
+!!! note
+    It is possible to automatically perform [text regions segmentation](predict.md#predict-the-segmentation) with a machine learning model.
 
-The segment editing panel allows you to perform several essential operations:
+## Text lines segmentation
 
-- Drawing baselines corresponding to the locations of the text on the image in two ways:
-    - Point by point plot;
-    - Free route (not recommended).
-- Adding points on a line (select it then double-click at the point’s location);
-- Moving one or more point(s) by a simple select-drag;
-- Deleting one or more point(s) on a line;
-- Cutting one or more lines using the scissors tool;
-- Joining one or more lines;
-- Configuring the reading direction of a line;
-- Activating the calculation of the polygons associated with each line (this task is automatic and managed asynchronously without any action on the part of the user after the first use).
+Text lines segmentation is a mandatory process in eScriptorium in order to transcribe a text. In eScriptorium, text lines are composed with two elements:
 
-In this panel, it is also possible to create regions (or zones ) and associate segments to them. Note that a segment located within the perimeter of a region is not always automatically associated to it.
+![image: Illustration of a baseline and its mask](img/segment/baseline.png "Illustration of a baseline and its mask.")
 
-All the operations that can be carried out in this panel are detailed in a help window (to display it, click on “?”). It will thus be noted that there are many shortcuts making it possible to simplify all these operations.
+- a **baseline**, or a **topline** (here in red), which is a virtual line passing through at least two coordinate points. It is on this line that the text is written, or from which it is hanging.
 
-!!! Note
-    The following is a copy of the content of the contextual window appearing when clicking on "?".
+- a **mask** (here in purple), which is a polygon defined by at least three coordinates points. It delimits the area of pixels containing the text of the baseline or topline, and is calculated automatically by eScriptorium based on its coordinates.
```

Comment:
I think "the coordinates" at the end would be better because here the "its" seems to be linked to eSc for me